R/count-operators.R

```diff
@@ -0,0 +1,924 @@
+#' @rdname CountOperators
+#' @title Checking occurrences or counts of elements in inputs
+#' @description \code{Count} gives the total count of elements that satisfy some pre-specified conditions
+#'  in the case of a single input and the count produced element wise for multiple inputs.
```

Comment:
Agree about the elementwise to element-wise change. Not so sure about the by case change since that usually means rows. Updated in the last commit in  453f10f